verify.py

```diff
@@ -79,14 +79,15 @@ def get_directory_contents(localpath):
 #============================================================================
 
 class Config():
-    '''Object representing the options from import/export config and 
+    '''Object representing the options from import/export config and
        command-line arguments.'''
     def __init__(self, configfile, auth):
-        print("Loading configuration options from import/export config file...")
+        print('''Loading configuration options from import/export config file
```

Comment:
Python will print the line break inside the docstring, so the formatting of this line gets messed up.

An alternative is to concatenate long strings implicitly:

```
print("Loading configuration options from import/export config file",
        "'{0}'...".format(configfile))
```

I hope these comments are helpful and not annoying.